app/javascript/mixins/mixin-scroll-magic.js

```diff
@@ -49,7 +49,8 @@ export default {
       let navScrollMagic = new ScrollMagic.Controller()
       let scrollMagicScenes = []
 
-      // add scene for each item in the nav
+      // add scene for each item in the nav, plus '#section-our-work'
+      this.pages.push({ id: 'our-work' })
       this.pages.forEach(link => {
         let sideNav = {},
           stickyBar = {}
```

Comment:
For easier keyboard editing (cut/paste lines), define each variable on its own line without shortcuts. It's slightly easier to scan through the code as well that way

Reply:
Whoops, didn't notice this was not your code --- in any case, this shortcut gets in the way of scanning, quick reordering and is slightly more prone  to syntax errors

Reply:
Yep, I'm not personally in the habit of initialising variables like this but I've made the change anyway. 👍🏻